studio/app/optinist/wrappers/caiman/params/caiman_cnmf.yaml

```diff
@@ -1,11 +1,30 @@
-p: 1
```

Comment:
@quanpython 
As a quick point, please configure cnmf and cnmfe with separate nodes.

However, as you are currently responding to this issue, many codes are common between cnmf and cnmfe, so please take into account the commonality of codes even when the nodes are separated.

Reply:
@itutu-tienday
I implemented it.